akka-http-core/src/main/java/akka/http/javadsl/model/headers/SameSite.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model.headers;
+
+/**
+ * The Cookie SameSite attribute as defined by <a href="https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00">RFC6265bis</a>
+ * and <a href="https://tools.ietf.org/html/draft-west-cookie-incrementalism-00">Incrementally Better Cookies</a>.
+ */
+public enum SameSite {
+    Strict,
+    Lax,
+    // SameSite.None is different from not adding the SameSite attribute in a cookie.
```

Comment:
Should we make this a proper javadoc comment? Could you also add the link to the Chrome post here if that's the best reference we currently have?

Reply:
I'm not sure we should be the ones explaining how `SameSite=None` works, and adding javadoc to `None` implies that we should do the same for `Lax` and `Strict`(?). That is why it is a code comment instead. I briefly discussed that with @raboof as well and we decided that a code comment for maintainers would suffice. 

Regarding the reference, I've added this: https://github.com/akka/akka-http/pull/2928/files#diff-f4b4205d613cfb57115bc38c5c78060eR9

Reply:
I see, fine with me.